index.html

```diff
@@ -21,7 +21,9 @@ <h1 class="hero text-3xl font-bold bg-base-200">
             <div style="width:50vw" class="text-center text-balance">
                 <h1 class="text-4xl font-bold">Meditative Tea House Management Game</h1>
                 <p class="py-6">From harvesting tea leaves to brewing the perfect cup of tea, run an independent tea house your way in Tea Time. Currently in Alpha Development.</p>
-                <button class="btn btn-primary btn-wide">Join Mailing List</button>
+                <a class="mailtoui" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">
+                    <button class="btn btn-primary btn-wide">Request Discord Access</button>
```

Comment:
Can we add this Discord icon to the button, to the left of the text?
[OpenMoji Discord icon](https://openmoji.org/library/emoji-E061/)

Reply:
Sure thing! Added!